tests/fake_cgw_client.py

```diff
@@ -0,0 +1,213 @@
+from pubtools._content_gateway.cgw_client import CGWClientError
+import six
+import inspect
+
+
+def tracker(f):
+    print(f)
```

Comment:
I don't think even the debug is needed there just because this class will only get called for testing purposes and during the test it ignores all the debug messages. So removing this print message completely.